### DISCUSSION TITLE
Still getting declined over and over.

### DISCUSSION THREAD

**Discussion by Iced-Coded** [community/community#54169] [GitHub Education]
### Hi everyone,

Hello! So, I've sent like 7-8 applications, and 3-4 of them were sent at the end of the previous month, so, firstly, I've uploaded translated student ID (because they've requested that from me before), then, I've got declined because I needed to use my PC camera to upload image, and that's okay, but then! I've got literally no reasons, just 
"Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved."
I mean, I have my student ID until 31.05.2024, **but!** I can't make a new one until this one expires, I don't have affiliation documents (they're at the principal's office or only my parents have them.).

**Comment by samus-aran** [comment]
HI @Iced-Coded 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.
